src/main/com/yetanalytics/datasim/input.clj

```diff
@@ -18,7 +18,9 @@
 (s/def ::personae
   ::personae/personae)
 
-(s/def ::alignments ::alignments/alignments)
+(s/def ::alignments
+  (s/every ::alignments/actor-alignment
+           :into []))
```

Comment:
note that `:into` only provides a collection for generation.. if you're trying to do a type predicate it would be like `:kind vector?` though I've recently leaned towards just using `every` w/o any args